### PR TITLE
feat: allow multiple quote reply languages

### PR DESCRIPTION
**Pull request by rubenmoya**
This PR allows to handle multiple languages when parsing reply quote headers

### CODE REVIEW

**Review by mariorn87 — APPROVED** [review]
🙌

File: lib/email_reply_parser.rb [hunk 1]
```diff
@@ -134,6 +138,33 @@ def read(text)
     EMPTY = "".freeze
     SIGNATURE = '(?m)(--\s*$|__\s*$|\w-$)|(^(\w+\s+){1,3}ym morf tneS$)'
```
Comment (leio10) [inline lib/email_reply_parser.rb]: :speech_balloon: No multilanguage for signature detection?

File: lib/email_reply_parser.rb [hunk 2]
```diff
@@ -134,6 +138,33 @@ def read(text)
...
+      /\s*El.*a las.*escribi.*\s*:$/,
+      /^(De|Enviado|Para|Asunto):.*$/m,
+      # French
+      /Le.*a .*crit.*[> ]:$/m,
+      /^(De|A|Envoyé|Objet)\s*:.*$/m,
+      # German
+      /[a-zA-Z]{2,5}.*schrieb.*:$/,
+      # Italian
+      /^Il.+?ha scritto:$/
+    ]
```
Comment (leio10) [inline lib/email_reply_parser.rb]: :speech_balloon: You can freeze all these constants

**Review by leio10 — APPROVED** [review]
Awesome!! :tada:

File: lib/email_reply_parser.rb (hunk 2 above)
Comment (rubenmoya) [reply lib/email_reply_parser.rb]: Done!

File: lib/email_reply_parser.rb (hunk 1 above)
Comment (rubenmoya) [reply lib/email_reply_parser.rb]: Not yet, we can keep improving it in the future

**Pull request merged**